lib/rules/jsx-sort-props.js

```diff
@@ -14,6 +14,19 @@ function isCallbackPropName(name) {
   return /^on[A-Z]/.test(name);
 }
 
+var RESEVED_PROPS_LIST = [
+  'children',
+  'dangerouslySetInnerHTML',
+  'htmlFor',
+  'key',
+  'ref',
+  'style'
+];
+
+function isReservedPropName(name, list) {
```

Comment:
I can see an argument for allowing a custom sorting of the reserved list - but by calling it "reserved", that means it's not something the user can change. In other words, "foo" isn't considered "reserved" by React, thus it should *never* be considered "reserved" by anything else.